components/RatingsTable/index.tsx

```diff
@@ -132,6 +133,15 @@ Props): JSX.Element {
     updateFilter(event.target.value);
   }, [setSearchText, updateFilter]);
 
+  const getPlayerInfo = ((name) => {
```

Comment:
needed to get information to populate the Link to the player page

Reply:
There has to be a better way to do this. If we go in the route I described in my review about the issue with linking the entire row, we should be able to render without even knowing who the player is. No more O(n^2) hell 😄

Reply:
corrected